### ISSUE TITLE
[BUG] Illegal reflective access by com.griefdefender.util.BootstrapUtil

### ISSUE THREAD

**Issue by mmolitor87** [bloodmc/GriefDefender#382]
- **Versions**
Spigot/Paper/Sponge: git-Paper-529 (MC: 1.16.5)
GriefDefender: GriefDefender 1.5.11-DEV
LuckPerms: LuckPerms v5.3.0
Java: openjdk version "11.0.10" 2021-01-19

- **Describe the bug**

During server startup the following is in the console:

```
[21:55:47 INFO]: [GriefDefender] Enabling GriefDefender v1.5.11-DEV
[21:55:47 INFO]: [GriefDefender] Loading libraries...
> WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.griefdefender.util.BootstrapUtil (file:/opt/minecraft/plugins/griefdefender-bukkit-1.5.11-DEV.jar) to method java.net.URLClassLoader.addURL(java.net.URL)
WARNING: Please consider reporting this to the maintainers of com.griefdefender.util.BootstrapUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[21:55:47 INFO]: [GriefDefender] GriefDefender boot start.
[21:55:47 INFO]: [GriefDefender] Loading configuration...
[21:55:48 INFO]: [GriefDefender] [ACF] Enabled Asynchronous Tab Completion Support!
[21:55:48 INFO]: [GriefDefender] Loaded successfully.
```

- **Steps to reproduce the behavior**

Start the server.

- **Expected behavior**

No errors.

- **GDDEBUG**

- **Screenshots**

**Comment by catronomix** [comment]
I have the same error.

```
[12:58:25 INFO]: [GriefDefender] Loading libraries...
> WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.griefdefender.util.BootstrapUtil (file:/*****/*****/*****/plugins/griefdefender-1.5.9.jar) to method java.net.URLClassLoader.addURL(java.net.URL)
WARNING: Please consider reporting this to the maintainers of com.griefdefender.util.BootstrapUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[12:58:25 INFO]: [GriefDefender] GriefDefender boot start.
```

GriefDefender 1.5.9
running  Paper version git-Paper-640 (MC: 1.16.5) 
OpenJDK 64-Bit Server VM (build 11.0.6+10-post-Debian-1bpo91, mixed mode, sharing)

**Comment by bloodmc** [comment]
Fixed in 2.0
